mlflow/gateway/schemas/chat.py

```diff
@@ -1,28 +1,28 @@
 from enum import Enum
 from typing import List, Optional
 
-from pydantic import BaseModel, Extra, Field
+from pydantic import Field
 
+from ..base_models import RequestModel, ResponseModel
 from ..config import RouteType
 
 
-class Message(BaseModel):
```

Comment:
Split this into request message and response message. Request message allows extra fields, response message ignores them